### DISCUSSION TITLE
Ignore uncertainty & variability in simulation using nlmixr fit object

### DISCUSSION THREAD

**Discussion by varungoelPMX** [nlmixrdevelopment/nlmixr#534] [Q&A]
I am using function "simulate" using nlmixr model fit object. Is there a way to control uncertainty, random, & IIV specification so these could be ignored for simulation?

**Comment by mattfidler** [comment]
You can use a zero omega matrix and a zero sigma matrix to control the variability currently.

You can see all the simulation infroemation in the fit by fit$simInfo

Hopefully this helps
